src/library.rs

```diff
@@ -192,20 +215,25 @@ pub struct Alias {
     pub c_identifier: String,
     pub typ: TypeId,
     pub target_c_type: String,
+    pub doc: Option<String>,
```

Comment:
I'm still thinking about how I could generate it (I mean, I need to check if I can use `#[deprecated]`, otherwise I'll just make something like: "!!!!!!!!!!!! DEPRECATED STUFF !!!!!!!!!!!!!!!!!!!!!!!!".